### PR TITLE
SDK-2230 Expose digital identity client example

### PR DESCRIPTION
**Pull request by laurent-yoti**
Added basic example, `profile` and `identity-profile`.

### CODE REVIEW

File: examples/digital-identity/views/pages/share.ejs [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          const { Yoti } = window
+          if (Yoti) {
+            await Yoti.createWebShare({
+              name: 'share example basic attributes',
+              domId: 'yoti-share-button-basic-attributes',
+              sdkId: "<%= yotiClientSdkId %>",
+              hooks: {
+                sessionIdResolver,
+                completionHandler,
+                errorListener,
```
Comment (saurabh-yoti) [inline examples/digital-identity/views/pages/share.ejs:145]: Both `sessionIdResolver` and `errorListener` triggered on Yoti modal click. Unsure why this happens. Check the browser developer tools console. Should see something like this:
```
sessionIdResolver found id ss.v2.ChZQOUhTWlFCZFNaVzQ5VnFkYVJXVlFnEgkzLmxkNS5nYnI=
Session errored - received errorCode: undefined
```

**Review by saurabh-yoti — COMMENTED** [review]
Minor things need to be addressed. Everything else LGTM!

File: examples/digital-identity/views/pages/share.ejs [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+          <ul>
+            <li><a id="yoti-profile-link-with-session-id" href="/">Go to profile using the session ID</a></li>
+            <li><a id="yoti-profile-link-with-receipt-id" href="/">Go to profile using the receipt ID</a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+  </main>
+<!--  <script src="https://www.yoti.com/share/client/v2"></script>-->
+  <script src="https://www.public.stg1.dmz.yoti.com/share/client/v2"></script>
```
[the remark this answers is not in the document]
Comment (laurent-yoti) [reply examples/digital-identity/views/pages/share.ejs]: 👍

File: examples/digital-identity/views/pages/share.ejs [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+      </div>
+
+      <div id="yoti-share-completion" class="yoti-sdk-integration-section hidden">
+        <h5>Share is completed!</h5>
+        <hr/>
+        <p>The <em>completionHandler</em> received the receipt ID corresponding to the share session completion.</p>
+        <p>
+          In this example, we are fetching a summary of the receipt data and display them below.
+          <br/>To do so, we are calling the server endpoint <code>GET /get-new-session-id</code> and pass the receipt ID as query params.
```
[the remark this answers is not in the document]
Comment (laurent-yoti) [reply examples/digital-identity/views/pages/share.ejs]: 👍

File: examples/digital-identity/views/pages/share.ejs (hunk 1 above)
Comment (laurent-yoti) [reply examples/digital-identity/views/pages/share.ejs:145]: We will get rid of the error raised with no `errorCode` soon, but also that `errorListener` might be removed in next version, so I added a comment in the code.

**Review by saurabh-yoti — APPROVED** [review]

**Pull request merged**